Sources/Foundation/FileManager+Win32.swift

```diff
@@ -412,6 +412,11 @@ extension FileManager {
         }
         return substitutePath
     }
+    
+    internal func _recursiveDestinationOfSymbolicLink(atPath path: String) throws -> String {
+        // TODO: Add recursive resolution.
+        return _destinationOfSymbolicLink(atPath: path)
```

Comment:
@compnerd This may require some changes on the Windows implementation to pass the new tests.

Reply:
Could you please take a look at that?  I can trigger a run if you like.

Reply:
I think we can just forward the call to `_canonicalizedPath(toFileAtPath:)` as it already recursively resolves the symlinks using `GetFullPathName`. Can you trigger a run, please?

Reply:
https://dev.azure.com/compnerd/swift-build/_build/results?buildId=28151&view=results